connectors/es/sink.py

```diff
@@ -356,33 +385,48 @@ async def _run(self):
             batch_num = 0
 
             while True:
+                force_flush = False
                 batch_num += 1
                 doc_size, doc = await self.fetch_doc()
-                if doc in (END_DOCS, EXTRACTOR_ERROR):
+                if isinstance(doc, ForceFlushSignal):
+                    self._logger.debug("Found ForceFlushSignal")
```

Comment:
```suggestion
                    self._logger.debug("Received flush signal")
```
slightly less alarming, IMO. Just a suggestion.